src/pages/ContentScripts/features/developer-hovercard-info/index.tsx

```diff
@@ -0,0 +1,84 @@
+import features from '../../../../feature-manager';
+import { getOpenrank } from '../../../../api/developer';
+import elementReady from 'element-ready';
+import { renderOpenRank } from './view';
+
+const featureId = features.getFeatureID(import.meta.url);
+
+const getDeveloperLatestOpenrank = async (developerName: string): Promise<string | null> => {
+  const data = await getOpenrank(developerName);
+  if (data) {
+    const values = Object.values(data) as string[];
+    const latestValue = values[values.length - 1];
+    return latestValue;
+  }
+  return null;
+};
+
+const getDeveloperName = (target: HTMLElement): string | null => {
+  const hovercardUrlAttribute = target.getAttribute('data-hovercard-url');
+  if (!hovercardUrlAttribute) return null;
+
+  const matches = hovercardUrlAttribute.match(/\/users\/([^\/]+)\/hovercard/);
+  return matches ? matches[1] : null;
+};
+
+const init = async (): Promise<void> => {
+  const hovercardSelector = '[data-hovercard-type="user"]';
+  document.querySelectorAll(hovercardSelector).forEach((element) => {
+    // isProcessing is used to Prevent OpenRank from adding duplicates
+    let isProcessing = false;
+
+    element.addEventListener('mouseover', async () => {
+      if (isProcessing) {
+        return;
+      }
+      isProcessing = true;
+
+      const developerName = getDeveloperName(element as HTMLElement) as string;
+
+      // Create a unique identifier for the popover
+      const popoverId = `popover-${developerName}`;
+
+      // Get the floating card container
+      const $popoverContainer =
+        'body > div.logged-in.env-production.page-responsive > div.Popover.js-hovercard-content.position-absolute > div > div > div';
+      const popover = await elementReady($popoverContainer, { stopOnDomReady: false });
+
+      const openRankDiv = popover?.querySelector('.openrank-info-container');
+      if (openRankDiv) {
+        const existingDeveloperName = openRankDiv.getAttribute('data-developer-name');
+        if (existingDeveloperName === developerName) {
+          return;
+        } else {
+          openRankDiv.remove();
+        }
+      }
```

Comment:
What does this block of code do?

Reply:
Also, can we simplify this method? maybe use `openrankDiv?.getAttribute()`. Something like the following (not tested):

``` typescript
      const openRankDiv = popover?.querySelector('.openrank-info-container');
      const existingDeveloperName = openRankDiv?.getAttribute('data-developer-name');
        if (existingDeveloperName === developerName) {
          return;
        }
        openRankDiv?.remove();
```

Reply:
already edited